app/assets/stylesheets/aleph.scss

```diff
@@ -0,0 +1,11 @@
+// Place all the styles related to the aleph controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.fa-check {
+  color: green;
```

Comment:
Additional commit addresses this. We'll now have to (get to?) explicitly include each css/scss file in the order that makes sense.